www/js/splash/startprefs.js

```diff
@@ -186,7 +186,7 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
             return 'root.main.goals';
           } else if ($rootScope.displayingIncident == true) {
             $rootScope.displayingIncident = false;
-            return 'root.main.diary';
```

Comment:
Add a `Logger.log` message here so that you know when it is executed? Or is there already a log statement that you plan to use instead to know which of the two code paths won the race?